nanopub/setup_profile.py

```diff
@@ -26,9 +26,7 @@ def validate_orcid_id(ctx, orcid_id: str):
     Check if valid ORCID iD, should be https://orcid.org/ + 16 digit in form:
         https://orcid.org/0000-0000-0000-0000
     """
-    if not orcid_id:
-        return None
-    elif re.match(ORCID_ID_REGEX, orcid_id):
+    if re.match(ORCID_ID_REGEX, orcid_id):
         return orcid_id
     else:
         raise ValueError('Your ORCID iD is not valid, please provide a valid ORCID iD that '
```

Comment:
I think this error should reiterate that it must look like ```https://orcid.org/0000-0000-0000-0000```. Even though it does indeed say that in the prompt, people will no doubt run this non-interactively, and if you don't see the help message for ```--orcid_id``` then it would be reasonable to input ```0000-0000-0000-0000``` or some other variation.

Reply:
It actually says so in the next line: https://github.com/fair-workflows/nanopub/blob/3d43ca79a35dcddcb23076bed7c6303266d8dc75/nanopub/setup_profile.py#L33